### DISCUSSION TITLE
Is it necessary to require php 8.2? can envy be enabled in php 8.1?

### DISCUSSION THREAD

**Discussion by WatheqAlshowaiter** [worksome/envy#33] [Q&A]
I have a project with php 8.1 and can not use envy because of this.

**Comment by MyeshaSteltzerZT1** [comment]
! I understand that you have a project that uses PHP 8.1 and you are unable to use Envy because of this. Have you considered using other environment variable libraries that support PHP 8.1? There may be other options available that could work for your project.
